rust-sdk/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "eppo"
+version = "0.2.0"
```

Comment:
```suggestion
version = "3.0.0"
```

Is that reasonable?

Reply:
Shall we go with `1.0.0` instead? There'll probably be many opportunities to bump major version, so aligning on `3.0.0` will quickly stop making sense.

Reply:
Another note that I forgot to mention is that rust-sdk does not support bandits yet. (It's in eppo-core but not rust-sdk.)